hack/build/internal/test/integration.go

```diff
@@ -32,23 +32,27 @@ func Integration(ctx context.Context, client *dagger.Client, base, flipt *dagger
 	}
 
 	var g errgroup.Group
-
-	for _, addr := range []string{
-		"grpc://flipt:9000",
-		"http://flipt:8080",
+	for _, namespace := range []string{
+		"",
+		"flags",
```

Comment:
nit: should we make this randomly generated? or perhaps something like `production` ?